### PR TITLE
Make doors equal

### PR DESCRIPTION
**Pull request by Eragonn1490**
Creates a standard that Just adds all the wooden doors together incase you just want to use a door made of wood

### CODE REVIEW

File: src/main/java/net/minecraftforge/oredict/OreDictionary.java [hunk 1]
```diff
@@ -106,6 +106,12 @@ private static void initVanillaEntries()
             registerOre("fenceGateWood", Blocks.JUNGLE_FENCE_GATE);
             registerOre("fenceGateWood", Blocks.DARK_OAK_FENCE_GATE);
             registerOre("fenceGateWood", Blocks.ACACIA_FENCE_GATE);
+            OreDictionary.registerOre("doorWood", new ItemStack(Items.ACACIA_DOOR));
```
Comment (SquidDev) [inline src/main/java/net/minecraftforge/oredict/OreDictionary.java]: Would it be better to do `registerOre("doorWood", Items.ACACIA_DOOR)`? This way it's consistent with the rest of the file.

Comment (Eragonn1490) [reply src/main/java/net/minecraftforge/oredict/OreDictionary.java]: you are right, that was my bad

**Review by Eragonn1490 — COMMENTED** [review]
fixed

File: src/main/java/net/minecraftforge/oredict/OreDictionary.java (hunk 1 above)
Comment (SquidDev) [reply src/main/java/net/minecraftforge/oredict/OreDictionary.java]: You can also drop the `ItemStack` constructor - while it creates one behind the scenes, it's a wee bit cleaner to read.

File: src/main/java/net/minecraftforge/oredict/OreDictionary.java [hunk 2]
```diff
@@ -106,6 +106,12 @@ private static void initVanillaEntries()
             registerOre("fenceGateWood", Blocks.JUNGLE_FENCE_GATE);
             registerOre("fenceGateWood", Blocks.DARK_OAK_FENCE_GATE);
             registerOre("fenceGateWood", Blocks.ACACIA_FENCE_GATE);
+            registerOre("doorWood", new ItemStack(Items.ACACIA_DOOR));
+            registerOre("doorWood", new ItemStack(Items.BIRCH_DOOR));
+            registerOre("doorWood", new ItemStack(Items.DARK_OAK_DOOR));
+            registerOre("doorWood", new ItemStack(Items.OAK_DOOR));
+            registerOre("doorWood", new ItemStack(Items.JUNGLE_DOOR));
+            registerOre("doorWood", new ItemStack(Items.SPRUCE_DOOR));
```
Comment (liach) [inline src/main/java/net/minecraftforge/oredict/OreDictionary.java]: Hmm, why do you wrap these items in item stacks while you can pass in the item as an argument?

**Comment by Eragonn1490** [comment]
@liach should be good now

**Comment by KnightMiner** [comment]
Can't you just oredict the vanilla wooden doors in your mod? No vanilla recipes use them, so it does not change that much to be in Forge code.

**Comment by ChloeDawn** [comment]
@KnightMiner Many things registered to the OreDictionary by default aren't used by vanilla. The point is to support a unified name. Having Forge provide it for wooden doors means other mods can depend on that ore name and append to it, without having any additional concerns about duplicate names or miscommunication of usages.

**Comment by LexManos** [comment]
Post a log from before and after. it has some important debug info to verify this doesnt break things.

**Comment by Eragonn1490** [comment]
@mezz 

all good on my end , I couldn't get the logs to upload to gist it kept having a script error. but the logs that lex wanted are here

https://www.dropbox.com/sh/w8hawbp9cco9u29/AAD-VGw32xqSBXXcPfT-owh0a?dl=0

**Pull request merged**